data/config/variety.conf

```diff
@@ -105,6 +105,12 @@ lightness_mode = 0
 min_rating_enabled = False
 min_rating = 4
 
+# Use a name regex filter?
+# name_regex_enabled = <True or False>
+# name_regex = <regex>
+name_regex_enabled = False
+name_regex = *
```

Comment:
nit: the default regex should probably be `.*` instead of `*`, as `*` alone isn't a valid regex

Reply:
Indeed, I updated the defaults to be a valid `.*`.
Thanks for spotting it.